connect/runtime/src/test/java/org/apache/kafka/connect/integration/ErrorHandlingIntegrationTest.java

```diff
@@ -75,9 +77,9 @@ public class ErrorHandlingIntegrationTest {
     private static final String DLQ_TOPIC = "my-connector-errors";
     private static final String CONNECTOR_NAME = "error-conn";
     private static final String TASK_ID = "error-conn-0";
-    private static final int NUM_RECORDS_PRODUCED = 20;
-    private static final int EXPECTED_CORRECT_RECORDS = 19;
+    private static final int NUM_RECORDS_PRODUCED = 1000;
     private static final int EXPECTED_INCORRECT_RECORDS = 1;
+    private static final int EXPECTED_CORRECT_RECORDS = NUM_RECORDS_PRODUCED - EXPECTED_INCORRECT_RECORDS;
```

Comment:
Again, I know this isn't your fault, but I'm a little confused at the use of this field in the `testErrantRecordReporter` case. I believe it can be replaced with `NUM_RECORDS_PRODUCED` [here](https://github.com/gharris1727/kafka/blob/f5845038014f3df29d505eeadd01403e9756728f/connect/runtime/src/test/java/org/apache/kafka/connect/integration/ErrorHandlingIntegrationTest.java#L220), since this case doesn't cover any errors that should prevent records from reaching tasks.

Reply:
Good catch, this test had the same flaw in two places, and I already fixed the other one. I think one test was copied from the other in this class, and these situational constants were used incorretly. I changed the constants so that the failure count is clearly relevant to the FaultyPassthrough, and may be less likely to be copied in the future.